src/components/AvatarGroup/AvatarGroup.st.css

```diff
@@ -156,19 +156,16 @@
     margin-left: 8px;
 }
 
-:size(large) .textButton,
-:size(medium) .textButton,
-:size(small) .textButton {
+.textButton {
     -st-mixin: TextButton(
-       MainTextColor '"--TextButtonColor"',
-       MainTextFont '"--TextButtonFont"'
+       MainTextColor value(TextButtonColor),
+       MainTextFont value(TextButtonFont)
     );
 }
 
-:size(xSmall) .textButton,
-:size(xxSmall) .textButton {
+.smallTextButton {
```

Comment:
Why add a class and not use the `state`?

Reply:
turns out starting with just :size doesn't work on processed css. I added .root now before it, and it's ok. So changed back to original classes, looks clean now.